features/polyExplorer/src/screens/entityDetails/companyRevenueChart/companyRevenueChart.css

```diff
@@ -29,31 +30,34 @@
 
 .revenue-chart-container .y-axis {
     position: absolute;
+    font-family: "Jost Medium";
```

Comment:
Why not simply put it in .revenue-chart-container? Or is there another font used somewhere else?